components/reactjs-components/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@crystallize/reactjs-components",
+    "license": "MIT",
+    "version": "0.1.0",
+    "author": "Crystallize <hello@crystallize.com> (https://crystallize.com)",
+    "contributors": [
+        "Håkon Krogh <hakon@crystallize.com>",
+        "Sébastien Morel <sebastien@crystallize.com>"
+    ],
+    "scripts": {
+        "watch": "yarn tsc -W",
+        "build": "yarn tsc",
```

Comment:
Just a note you don't need `yarn` here, `tsc` is fine by itself 🙂

Reply:
thanks!